File: Software Technologies - Exams/Kanban_Board_CU/Java-Skeleton/src/main/java/teistermask/entity/Task.java
package teistermask.entity;

import javax.persistence.*;

@Entity
@Table(name = "tasks")
public class Task {
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

	@Column(length = 65535, columnDefinition = "text", nullable = false)
	private String title;

    @Column(nullable = false)
    private String status;

    public Task() {
    }

    public Task(String title, String status) {
        this.title = title;
        this.status = status;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: Software Technologies - Exams/Kanban_Board_CU/JavaScript-Skeleton/models/Task.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Task = sequelize.define('Task', {
       title : {
           type: Sequelize.TEXT,
           allowNull: false
       },
       status : {
           type: Sequelize.STRING,
           allowNull: false
       }
    }, {
        timestamps : false
    });
    return Task;
};